<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>631</width>
      <height>551</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Song Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>HC(%)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>T(st)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>THC(%)</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="music_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Music Folder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="analyze_button">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Analyze</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_path_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>611</width>
      <height>21</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_path_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>611</width>
      <height>21</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_print1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>411</width>
      <height>16</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_print2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>411</width>
      <height>16</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
